.github/workflows/reusable_compatibility.yml

```diff
@@ -0,0 +1,134 @@
+# TODO
+name: Compatibility 
+
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: Check backward compatibility with this tag
+        type: string
+        default: "0.9.0"
+
+permissions:
+  contents: read
+
+jobs:
+  ubuntu-build:
+    name: Ubuntu
+    strategy:
+      matrix:
+        # TODO other systems?
+        os: ['ubuntu-22.04']
+        build_type: [Debug]
+        compiler: [{c: gcc, cxx: g++}]
+        shared_library: ['ON']
+        level_zero_provider: ['ON']
+        cuda_provider: ['ON']
+        install_tbb: ['ON']
+    runs-on: ${{matrix.os}}
+
+    steps:
+      # NOTE: we need jemalloc for older version of UMF
+    - name: Install apt packages
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y clang cmake libnuma-dev libjemalloc-dev 
+
+    - name: Install TBB apt package
+      if: matrix.install_tbb == 'ON'
+      run: |
+        sudo apt-get install -y libtbb-dev
+
+    - name: Checkout "tag" UMF version
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        fetch-depth: 0
+        ref: refs/tags/${{inputs.tag}}
+        path: ${{github.workspace}}/tag_version
+
+    - name: Install libhwloc
+      working-directory: ${{github.workspace}}/tag_version
+      run: .github/scripts/install_hwloc.sh
+        
+    - name: Get "tag" UMF version
+      working-directory: ${{github.workspace}}/tag_version
+      run: |
+        VERSION=$(git describe --tags --abbrev=0 | grep -oP '\d+\.\d+\.\d+')
+        echo "UMF_VERSION=$VERSION" >> $GITHUB_ENV
```

Comment:
It is not used, so why adding it to the `GITHUB_ENV`?

Reply:
removed